src/main/webapp/list.html

```diff
@@ -0,0 +1,177 @@
+          <option value="state:mo">Missouri</option>
+          <option value="state:ms">Mississippi</option>
+          <option value="state:mt">Montana</option>
+          <option value="state:nc">North Carolina</option>
+          <option value="state:nd">North Dakota</option>
+          <option value="state:ne">Nebraska</option>
+          <option value="state:nh">New Hampshire</option>
+          <option value="state:nj">New Jersey</option>
+          <option value="state:nm">New Mexico</option>
+          <option value="state:nv">Nevada</option>
+          <option value="state:ny">New York</option>
+          <option value="state:oh">Ohio</option>
+          <option value="state:ok">Oklahoma</option>
+          <option value="state:or">Oregon</option>
+          <option value="state:pa">Pennsylvania</option>
+          <option value="state:ri">Rhode Island</option>
+          <option value="state:sc">South Carolina</option>
+          <option value="state:sd">South Dakota</option>
+          <option value="state:tn">Tennessee</option>
+          <option value="state:tx">Texas</option>
+          <option value="state:ut">Utah</option>
+          <option value="state:va">Virginia</option>
+          <option value="state:vt">Vermont</option>
+          <option value="state:wa">Washington</option>
+          <option value="state:wi">Wisconsin</option>
+          <option value="state:wv">West Virginia</option>
+          <option value="state:wy">Wyoming</option> 
+        </select>
+      </div>
+    </div>
+
+    <div class="container-fluid" id="election-list-content">
+      <h3>Elections coming up in Washington</h3>
+      <ul class="list-group list-group-flush" id="election-list">
+        <li class="list-group-item" id="election-item">
+          <div class="election-quick-details">
+            <p>Washington State Primary Election</p>
+            <p>August 4, 2020</p>
+          </div>
+          <div class="learn-more-button">Learn more</div>
+        </li>
+        <li class="list-group-item" id="election-item">
+          <div class="election-quick-details">
+            <p>Other Election Title</p>
+            <p>August 40, 2020</p>
+          </div>
+          <div class="learn-more-button">Learn more</div>
+        </li>
+      </ul>
+      <h3>Elections coming up in the United States</h3>
+      <ul class="list-group list-group-flush" id="election-list">
+        <li class="list-group-item" id="election-item">
+          <div class="election-quick-details">
+            <p>Washington State Primary Election</p>
+            <p>August 4, 2020</p>
+          </div>
+          <div class="learn-more-button">Learn more</div>
+        </li>
+        <li class="list-group-item" id="election-item">
+          <div class="election-quick-details">
+            <p>Other Election Title</p>
+            <p>August 40, 2020</p>
+          </div>
+          <button class="learn-more-button">Learn more</button>
```

Comment:
Is this also supposed to be a div instead of a button?

Reply:
Whoops, good catch